mlir/tools/mlir-tblgen/OpFormatGen.cpp

```diff
@@ -113,13 +113,13 @@ struct AttributeLikeVariable : public VariableElement {
     return isa<VariableElement>(fe) && classof(cast<VariableElement>(fe));
   }
 
-  /// Returns true if the variable is a UnitAttr or a UnitProperty.
+  /// Returns true if the variable is a UnitAttr or a UnitProp.
   bool isUnit() const {
     if (const auto *attr = dyn_cast<AttributeVariable>(this))
       return attr->getVar()->attr.getBaseAttr().getAttrDefName() == "UnitAttr";
     if (const auto *prop = dyn_cast<PropertyVariable>(this)) {
       return prop->getVar()->prop.getBaseProperty().getPropertyDefName() ==
-             "UnitProperty";
+             "UnitProp";
```

Comment:
Isn't this going to fail now for `UnitProperty`?

Reply:
Good point, fixed